.github/workflows/ci.yml

```diff
@@ -44,10 +66,24 @@ jobs:
           # start xvfb in the background
           sudo /usr/bin/Xvfb $DISPLAY -screen 0 1280x1024x24 &
 
-      - name: Install system dependencies (MacOS)
+      - name: Cache macOS
+        id: cache-macos
+        if: runner.os == 'macOS'
+        uses: actions/cache@v2
+        with:
+          path: ${{ github.workspace }}/ManimCache
```

Comment:
Why are you caching `${{ github.workspace }}/ManimCache` does brew install anything there?

Reply:
Oh, right. brew installs packages to `/usr/local/Cellar`. I was just exploring and figuring things out.

Reply:
I think you shouldn't cache that. Rather I would set up to download FFmpeg from https://evermeet.cx/ffmpeg/ and latex using TinyTeX(see how it is done on Windows), rather than BasicTeX. And cache all of them.